### PR TITLE
chore: v0.13.0

### PR DESCRIPTION
**Pull request by AverageHelper**
About time tbh, lemme see if I remember our release process 😅

### CODE REVIEW

**Comment by AverageHelper** [comment]
Waiting on this until #166 merges

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Nothing!
+
+## [0.13.0] - 2024-08-24
```
Comment (JstnMcBrd) [inline CHANGELOG.md:12]: Just rebase on main and update this date and LGTM

**Review by JstnMcBrd — APPROVED** [review]

**Comment by AverageHelper** [comment]
uh, looks like I did something wrong for the docker deploy... we still only have a `main` tag.....

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AverageHelper) [reply CHANGELOG.md:12]: Fixed in #191

**Pull request merged**